api4/post.go

```diff
@@ -479,6 +482,9 @@ func patchPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Updating the file_ids of a post is not a supported operation and will be ignored
+	post.FileIds = nil
```

Comment:
We also end up doing a `UpdatePost(_, false)` here.

Reply:
See above.